src/views/AnnualReport.vue

```diff
@@ -870,6 +870,7 @@ export default {
           this.haveChanges = false
         }).catch(async error => {
           if (error && error.response && error.response.status === PAYMENT_REQUIRED) {
+            this.haveChanges = false
```

Comment:
Please add comment explaining why this special case.
Also in other 2 components.
Also in Correction.vue?